src/json_spirit/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 add_library(json_spirit STATIC
   json_spirit_reader.cpp
   json_spirit_writer.cpp)
-target_link_libraries(json_spirit common_utf8)
+target_link_libraries(json_spirit common_utf8 Boost::thread)
```

Comment:
why would json_spirit need to link against `Boost::thread`?

Reply:
https://github.com/ceph/ceph/blob/v16.0.0/src/json_spirit/json_spirit_reader_template.h#L18

I managed to compile it without it though so I dropped it. Something might've changed in the meantime (I wrote this 8 months ago), same for other libraries.